src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -628,4 +634,27 @@ namespace AppInstaller::Utility
 
         return path.filename();
     }
+
+    std::vector<std::string> SplitIntoWords(std::string_view input)

```

Comment:
Word breaking can benefit from knowing the locale.  Since we often have it, it might be good to include it here.  It might also be incorrect in ARP, so that is worth considering (maybe the actual application is in that locale, but the ARP data is in English [or half English]).

If it looks like that is likely to be too complex or that we simply don't have enough test data to be able to understand the implications, it can be considered at a later date.  I do believe that ICU tries its best to determine locale from the input string, so it may be sufficient for now and best to wait until we find counterexamples.

Reply:
Looking into this. I'm not familiar with what we have in the ARP regarding locale. But it seems to behave well enough without specifying it, and I'm not sure if using it only in some places could cause inconsistency (e.g. using it for manifest strings but not for ARP strings if there is no locale data); so I think it can be done later.